packages/mdc-snackbar/README.md

```diff
@@ -149,10 +198,18 @@ The adapter for snackbars must provide the following functions, with correct sig
 | `removeClass(className: string) => void` | Removes a class from the root element. |
 | `setAriaHidden() => void` | Sets `aria-hidden="true"` on the root element. |
 | `unsetAriaHidden() => void` | Removes the `aria-hidden` attribute from the root element. |
-| `setMessageText(message: string) => void` | Set the text content of the message element. |
-| `setActionText(actionText: string) => void` | Set the text content of the action element. |
 | `setActionAriaHidden() => void` | Sets `aria-hidden="true"` on the action element. |
 | `unsetActionAriaHidden() => void` | Removes the `aria-hidden` attribute from the action element. |
+| `setActionText(actionText: string) => void` | Set the text content of the action element. |
+| `setMessageText(message: string) => void` | Set the text content of the message element. |
+| `setFocus() => void` | Sets focus on the action button. |
+| `visibilityIsHidden() => boolean` | Returns document.hidden property. |
+| `registerBlurHandler(handler: EventListener) => void` | Registers an event handler to be called when a `blur` event is triggered on the action button |
+| `deregisterBlurHandler(handler: EventListener) => void` | Deregisters a `blur` event handler from the actionButton |
+| `registerVisibilityChangeHandler(handler: EventListener) => void` | Registers an event handler to be called when a 'visibilitychange' event occurs |
+| `deregisterVisibilityChangeHandler(handler: EventListener) => void` | Deregisters an event handler to be called when a 'visibilitychange' event occurs |
+| `registerCapturedInteractionHandler(evtType: string, handler: EventListener) => void` | Registers an event handler to be called when the given event type is triggered on the `body` |
```

Comment:
I don't think so. What this name suggests is that we are doing something with an interaction that has previously been captured (albeit immediately after).

I agree we should be acknowledging that we are listening during the capture phase. :+1: